updatecli/updatecli.d/rancher/shell/prometheus-federator/shell.yaml

```diff
@@ -0,0 +1,132 @@
+---
+name: "Bump rancher/shell image version"
+scms:
+  shell:
+    kind: "github"
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      username: "{{ .github.username }}"
+      token: "{{ requiredEnv .github.token }}"
+      owner: "rancher"
+      repository: "shell"
+      branch: "master"
+  prometheus-federator:
+    kind: "github"
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      username: "{{ .github.username }}"
+      token: "{{ requiredEnv .github.token }}"
+      owner: "rancher"
+      repository: "prometheus-federator"
+      branch: "main"
+      directory: "tmpdir-prometheus-federator/prometheus-federator"
+  helm-project-operator:
+    kind: "github"
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      username: "{{ .github.username }}"
+      token: "{{ requiredEnv .github.token }}"
+      owner: "rancher"
+      repository: "helm-project-operator"
+      branch: "main"
+      directory: "tmpdir-prometheus-federator/helm-project-operator"
+
+pullrequests:
+  github:
+    title: "Bump rancher/shell image version"
+    kind: "github"
+    scmid: "prometheus-federator"
+    spec:
+      automerge: false
+
+sources:
+  source1:
```

Comment:
Is the sourceX and conditionX numbering here because of ordering? I assume it would need something more descriptive (I get there is a name but with `dependson` you still have no clue what it depends on).

Reply:
Yep a better name would be useful to understand the `depends_on`

Something to keep in mind is sources are not ordered hence the need for `depends_on`

Reply:
Good point. Will update those names.

Reply:
Updated.